scripts/README.md

```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
 You can also check that the schema properties reflect the Ontario theme. In the Core Selector dropdown menu, choose ckan, and then select Schema from the menu. The Unique Key Field should be `index_id`.
 
 ![image](https://user-images.githubusercontent.com/1254764/167931044-3bb4686a-eebd-4651-92ce-b6a82cb0309f.png)
+
+
+# Load Organization Data Script
+
+This script (`load_organization_data.py`) allows you to load organization data into your application using the CKAN API. Before running the script, please ensure that you have the necessary prerequisites and follow the instructions below.
+
+## Prerequisites
+
+- Python (version 3.x)
+- Requests library (can be installed via `pip install requests`)
+
+## Setup
+
+1. Clone the repository containing the script:
+
+2. Install the required dependencies by navigating to the repository directory and running:
+
+     pip install -r requirements.txt
+
+
+## Configuration
+
+1. Obtain an API key from your CKAN application. You can find the API key in the CKAN user settings at `http://localhost/user/edit/admin`. Make sure you are logged in as an administrator.
+
+2. Open the `load_organization_data.py` script in a text editor.
+
+3. Locate the `api_token` variable in the script and replace the empty string `""` with your obtained API key:
+```python
+api_token = "your_api_key_here"
+
+
+## Usage 
```

Comment:
the content in this section is all bullet-points/steps to follow. Make sure they are bullet points. Use the other section in the document as a reference for styling.